examples/util/openvino/OpenVINOModelImporter.js

```diff
@@ -681,6 +737,90 @@ class OpenVINOModelImporter {
 
           opCode = this._nn.SOFTMAX;
         } break;
+        case 'Normalize': {
+          const input = node.inputs[0];
+          const inDims = input.shape();
+          const inputId = this._getTensorId(input);
+          console.log(`  input shape: [${input.shape()}]`);
+
+          const acrossSpatial = node.getInt('across_spatial');
+          const channelShared = node.getInt('channel_shared');
+          const eps = node.getFloat('eps');
+
+          if (acrossSpatial !== 0 || channelShared !== 0) {
+            throw new Error(`Normalize not support across_spatial ${across_spatial} channel_shared ${channel_shared}.`);
+          }
+
+          inputs.push(inputId);
+
+          const output = node.outputs[0];
+
+          if (i === graph.nodes.length - 1) {
+            this._tensorIds[output.graphId()] = inputId;
+            console.log(`  skip last op: Normalize`);
+          } else {
+            const outDims = output.shape();
+            const outputType = {
+              type: this._getTypeCode(output.dataType()), dimensions: outDims
+            };
+            const outputId = this._addNamedOperand(output.graphId(), outputType);
+            outputs.push(outputId);
+            console.log(`  output shape: [${outDims}]`);
+
+            opCode = this._nn.L2_NORMALIZATION;
+          }
+        } break;
+        case 'PReLU': {
```

Comment:
should we handle `channel_shared` parameter of [PReLU](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_prepare_model_convert_model_IRLayersCatalogSpec.html#PReLU)?

Reply:
No, there are only tow inputs as input and slop in [NNAPI](https://developer.android.com/ndk/reference/group/neural-networks.html#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf), and there is auto-handle channel_shared parameter in NNAPI,  so we no need this parameter.

Reply:
Then, please add comment to say we don't support `channel_shared` parameter of OpenVINO's PReLU.